### PR TITLE
XENOPS-794 Gradle cleanup; Fixed javadoc

### PR DESCRIPTION
**Pull request by todorinskiz**

### CODE REVIEW

File: publish.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
+apply plugin: 'java-library'
```
Comment (vierbergenlars) [inline publish.gradle:3]: I believe that `java-library` is not compatible with the `java` plugin, so you have to pick one.

File: alfresco-zipkin-repo/build.gradle [hunk 2]
```diff
@@ -22,31 +22,6 @@ dependencies {
     testRuntime group: 'javax.servlet', name: 'servlet-api', version: '2.5'
 }
 
-amp {
-    from("$projectDir/src/main/resources"){
```
Comment (vierbergenlars) [inline alfresco-zipkin-repo/build.gradle:26]: You're not putting resources in the AMP anymore. Then please put the things that need to be in `src/main/amp`, where they will be picked up and placed in the amp automatically.
https://github.com/xenit-eu/alfresco-gradle-sdk#conventional-project-layout

**Review by vierbergenlars — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 3]
```diff
@@ -3,6 +3,8 @@ plugins {
     id 'org.ajoberstar.reckon' version "0.13.0"
 }
 
+version = "2.0.0"
```
Comment (vierbergenlars) [inline build.gradle:6]: :question: You can't set a version and use reckon at the same time.

File: alfresco-zipkin-repo/build.gradle (hunk 2 above)
Comment (todorinskiz) [reply alfresco-zipkin-repo/build.gradle:26]: Restructured the resources, preserved identical file locations as the original amp

File: build.gradle (hunk 3 above)
Comment (todorinskiz) [reply build.gradle]: committed per accident :upside_down_face:

File: publish.gradle (hunk 1 above)
Comment (todorinskiz) [reply publish.gradle]: removed!

File: alfresco-zipkin-repo/build.gradle [hunk 4]
```diff
@@ -22,29 +24,18 @@ dependencies {
...
-        include '**/*.properties'
-        expand( artifactId: "$project.name",
-                title: "$project.name",
-                description: "$description",
-                version: "$project.version")
-    }
-    from("$projectDir/src/main/resources"){
-        exclude '**/*.properties'
+project.tasks.amp.config {
+    from files("${project.projectDir}/src/main/amp/") {
```
Comment (vierbergenlars) [inline alfresco-zipkin-repo/build.gradle:28]: If you moved the files to be packaged to `src/main/amp/config` (as the conventional layout prescribes), there would be no need to add this `from()`

Comment (todorinskiz) [reply alfresco-zipkin-repo/build.gradle:28]: @vierbergenlars I preserved the AMP structure as it was. Moving it to config would change the location of the Solr configuration in the AMP, which I am not 100% sure where to place.
Still convinced we should move this?

**Comment by driesdeproost** [comment]
Share won't start in the composeUp setup

Due to:
Caused by: java.io.FileNotFoundException: class path resource [config/alfresco/web-extension/alfresco-global.properties] cannot be opened because it does not exist

The global properties seem to be present in the amp in the right location though.
I'm having a look

**Comment by vierbergenlars** [comment]
> The global properties seem to be present in the amp in the right location though.
I'm having a look
Stuff in `/config/` in the AMP is not installed in `/config/` in the WAR, see [AMP to WAR mapping](https://docs.alfresco.com/content-services/6.1/develop/extension-packaging/#defaultamp2warmap)

**Pull request closed without merging**